### DISCUSSION TITLE
Updatable Edge

### DISCUSSION THREAD

**Discussion by azamat7g** [bcakmakoglu/vue-flow#675] [Q&A] [answered]
Hi! I try to make updatable edge with delete edge on drop. I saw example on reactFlow(https://reactflow.dev/docs/examples/edges/delete-edge-on-drop/) and try to make it with vueFlow. Everything work fine except one case. If start dragging edge from node_1 and connect to node_1 (node not changed) edge removes. In reactFlow every time on edge updated calls `onEdgeUpdate`, in vueFlow `onEdgeUpdate` calls when edge connects to another node (if connect the same node, function not fire). Is it possible to fix? Or in vueFlow this things do with another way? 

Example code: https://codesandbox.io/p/sandbox/elated-cache-y32fjl

`VueFlow preview:`
<video src="https://user-images.githubusercontent.com/10195180/218461888-684fff84-8f1a-4db3-99db-557ba3b2c596.mov" data-canonical-src="https://user-images.githubusercontent.com/10195180/218461888-684fff84-8f1a-4db3-99db-557ba3b2c596.mov" controls="controls" muted="muted"></video>

`ReactFlow preview:`
<video src="https://user-images.githubusercontent.com/10195180/218461664-75d0f167-4744-4f79-b1ae-314bcefa67b7.mov" data-canonical-src="https://user-images.githubusercontent.com/10195180/218461664-75d0f167-4744-4f79-b1ae-314bcefa67b7.mov" controls="controls" muted="muted"></video>

**Comment by bcakmakoglu** [comment, marked as answer]
Mh... currently the behavior is to not emit `edgeUpdate` when the connection already exists, which just avoids updates that are unnecessary.
React Flow doesn't do this check and thus *always* emits an `edgeUpdate` event, even if it's the same edge.

I could change the current behavior to match that of React Flow.

**Reply by bcakmakoglu** [reply]
Change is released with `1.14.3`
